### DISCUSSION TITLE
BBC Stream Changes

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#249] [Tips]
These URL lists might be of interest to rAudio-1 users who are hearing the warning announcements on the BBC streams they use:  
https://gist.github.com/bpsib/67089b959e4fa898af69fea59ad74bc3

**Comment by ceejayemm** [comment]
I keep up to date with all UK radio station URLs via:

http://www.radiofeeds.co.uk/

At the moment this gives different URLs to those shown in the Github link from @R101-Github above.  I don't know which is the definitive list as the BBC don't seem to make their URLs generally available.

Chris

**Reply by R101G** [reply]
@ceejayemm

The only BBC info seems to be here:

https://www.bbc.co.uk/blogs/internet/entries/481e7233-0aea-4b15-8ace-878ce549108c

And as you can see, this notice is only meant for manufacturers, not listeners.

Any streams starting bbcmedia.ic.llnwd.net or bbcwssc.ic.llnwd.net will be closing.

**Reply by klassikmann** [reply]
GIve this list a try:
https://gist.github.com/bpsib/67089b959e4fa898af69fea59ad74bc3
